packages/aws-cdk-lib/aws-stepfunctions/lib/states/distributed-map.ts

```diff
@@ -234,9 +235,10 @@ export class DistributedMap extends MapBase implements INextable {
    */
   public toStateJson(): object {
     let rendered: any = super.toStateJson();
-    if (this.mapExecutionType) {
```

Comment:
Removed this conditional statement as it is meaningless, because in the following code, mapExecutionType is always set, making the condition always evaluate to true.

https://github.com/aws/aws-cdk/blob/a37bc7283d2937a43a6c19d8a66a3347167cef75/packages/aws-cdk-lib/aws-stepfunctions/lib/states/distributed-map.ts#L129